backend/src/main/java/fr/anettehaferkorn/backend/security/controller/CreateUserController.java

```diff
@@ -0,0 +1,27 @@
+package fr.anettehaferkorn.backend.security.controller;
+
+import fr.anettehaferkorn.backend.security.model.AppUserDTO;
+import fr.anettehaferkorn.backend.security.service.CreateUserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@RequestMapping("/auth/createUser")
+public class CreateUserController {
```

Comment:
Maybe better user controller. In principle the controller should be responsible for all user interactions like editing/deleting also even though might not intend to integrate them yet

Reply:
Or is it's function rather a signup? If so I would name it like this, or maybe rename the loginController to AuthController or something like this and let it handle signup and login